doc/rtd/tutorial/qemu.rst

```diff
@@ -144,6 +144,7 @@ Now let's run the following command, which creates a file named
     $ cat << EOF > meta-data
     instance-id: someid/somehostname
     local-hostname: jammy
+    create_hostname_file: true
```

Comment:
Same here. This would go into user data, not meta data. Though...I'm not sure it's worth mentioning hostnames at all.